packages/fast-layouts-react/src/column/column.spec.tsx

```diff
@@ -241,6 +242,53 @@ describe("Column", (): void => {
 
         expect(rendered.props().gutter).toBe(2);
     });
+
+    test("column should use a default breakpoint of zero when the DOM is unavailable and default breakpoint is unset", () => {
+        // make DOM unavailable for test
+        const windowSpy: jest.SpyInstance<any> = jest.spyOn(
+            global as any,
+            "window",
+            "get"
+        );
+        windowSpy.mockImplementation(() => ({ undefined }));
+        expect(canUseDOM()).toEqual(false);
```

Comment:
We had some issues with tests failing with this implementation. Can you check out the breakpoint tracker test updates in this commit and see if we can leverage that pattern here? https://github.com/microsoft/fast-dna/commit/25633f6e63c69a565fb65e654607b6a87b7fe9d2

Reply:
That's much cleaner as well, yep I'll update